docs/src/languages/php/tuning.md

```diff
@@ -30,24 +30,24 @@ The most common issue is not allowing the right cookies as part of the router ca
 Some cookies, such as session cookies, need to be allowed,
 whereas others, such as marketing and analytics cookies,
 usually shouldn't be allowed to be part of the cache key.
-See the [router cache](/configuration/routes/cache.md) documentation
-and the [cookie entry](/configuration/routes/cache.md#cookies).
+See the [router cache](../../configuration/routes/cache.md) documentation
+and the [cookie entry](../../configuration/routes/cache.md#cookies).
 
 You also need to ensure that your application is sending the correct `cache-control` header.
 The router cache obeys whatever cache headers your application sends,
 so send it good ones.
 
 Static assets cache headers are set using the `expires` key in `.platform.app.yaml`.
-See the [`web.locations`](/configuration/app/app-reference.md#locations) documentation for more details.
+See the [`web.locations`](../../configuration/app/app-reference.md#locations) documentation for more details.
```

Comment:
A detail too, but since the styling doesn't fully show that the [link is clickable](https://developers.google.com/style/link-text?hl=en), should the word `documentation` be part of the link as well?